src/QsCompiler/CommandLineTool/Commands/Build.cs

```diff
@@ -24,47 +23,70 @@ public static IEnumerable<Example> UsageExamples
+            [Option(
+                "response-files",
+                Required = true,
+                SetName = RESPONSE_FILES,
+                HelpText = "Response file(s) providing command arguments. Required only if no other arguments are specified. Non-default values for options specified via command line take precedence.")]
             public IEnumerable<string> ResponseFiles { get; set; }
 
-            [Option('o', "output", Required = false, SetName = CODE_MODE,
-            HelpText = "Destination folder where the output of the compilation will be generated.")]
+            [Option(
+                'o',
+                "output",
+                Required = false,
+                SetName = CODE_MODE,
+                HelpText = "Destination folder where the output of the compilation will be generated.")]
             public string OutputFolder { get; set; }
 
-            [Option("doc", Required = false, SetName = CODE_MODE,
-            HelpText = "Destination folder where documentation will be generated.")]
+            [Option(
+                "doc",
+                Required = false,
+                SetName = CODE_MODE,
+                HelpText = "Destination folder where documentation will be generated.")]
             public string DocFolder { get; set; }
 
-            [Option("proj", Required = false, SetName = CODE_MODE,
-            HelpText = "Name of the project (needs to be usable as file name).")]
+            [Option(
+                "proj",
+                Required = false,
+                SetName = CODE_MODE,
+                HelpText = "Name of the project (needs to be usable as file name).")]
             public string ProjectName { get; set; }
 
-            [Option("emit-dll", Required = false, Default = false, SetName = CODE_MODE,
-            HelpText = "Specifies whether the compiler should emit a .NET Core dll containing the compiled Q# code.")]
+            [Option(
+                "emit-dll",
+                Required = false,
+                Default = false,
+                SetName = CODE_MODE,
+                HelpText = "Specifies whether the compiler should emit a .NET Core dll containing the compiled Q# code.")]
             public bool EmitDll { get; set; }
 
-            [Option("perf", Required = false, SetName = CODE_MODE,
-            HelpText = "Destination folder where the output of the performance assessment will be generated.")]
+            [Option(
+                "perf",
+                Required = false,
+                SetName = CODE_MODE,
+                HelpText = "Destination folder where the output of the performance assessment will be generated.")]
             public string PerfFolder { get; set; }
 
-
             /// <summary>
-            /// Reads the content of all specified response files and processes it using FromResponseFiles. 
+            /// Reads the content of all specified response files and processes it using FromResponseFiles.
```

Comment:
I really appreciate this removing trailing whitespace!

Reply:
I am really tempted to set `dotnet_diagnostic.SA1028.severity = error`, which would make trailing whitespace a compile error. :)